chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/scenario/QpsClockConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package com.griddynamics.jagger.engine.e1.scenario;
+
+import com.griddynamics.jagger.util.SystemClock;
+
+import java.math.BigDecimal;
+
+/**
+ * Clock Configuration for Qps load
+ */
+public class QpsClockConfiguration extends AbstractRateClockConfiguration {
+
+    @Override
+    protected WorkloadClock getRateClock(int tickInterval, TpsRouter tpsRouter, WorkloadSuggestionMaker workloadSuggestionMaker, SystemClock systemClock, int maxThreadNumber) {
+
+        return new QpsClock(tickInterval, maxThreadNumber, createDesiredTps(BigDecimal.valueOf(getTps())));
+    }
+
+    @Override
+    public String toString() {
+        if (isRumpUp()){
+            return getTps() + " rump-up rps";
+        }
+        return getTps() + " rps";
```

Comment:
"rps" to see the same load explanation as for old implementation on WebUI and pdf report.